### PR TITLE
Cyton and Future Proofing Breaking Changes

### PR DESCRIPTION
**Pull request by aj-ptw**
# 2.0.0

### New Features
* `index.js` file allows for ES6 destructing

### Breaking Changes
* Change name of `OpenBCIBoard` to `Cyton` to follow new naming convention.
    Simply change:
    ```ecmascript 6
    const OpenBCIBoard = require('openbci').OpenBCIBoard;
    const ourBoard = new OpenBCIBoard();
    ```
    ```ecmascript 6
    const Cyton = require('openbci').Cyton;
    const ourBoard = new Cyton();
    ```
* Major change to how board is initialized with removal of `factory` paradigm.
* Drop support for Node 4 and 5 due to lack of EMACS 6

### Bug Fixes
* Documentation error with `testSignal` function.
 
### Enhancements
* Add more tests for public API functions.

### Todalos
- [ ] Fix timeSync.js example for new format.
- [ ] Test timeSync.js with real board.

### CODE REVIEW

File: openBCIConstants.js [hunk 1]
```diff
@@ -203,6 +203,7 @@ const obciSimulatorFragmentationNone = 'none';
 
 /** Possible Sample Rates */
 const obciSampleRate125 = 125;
+const obciSampleRate200 = 200;
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove this

File: openBCIConstants.js [hunk 2]
```diff
@@ -232,9 +233,13 @@ const obciByteStart = 0xA0;
 const obciByteStop = 0xC0;
 
 /** Errors */
+const errorNobleAlreadyScanning = 'Scan already under way';
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 3]
```diff
@@ -232,9 +233,13 @@ const obciByteStart = 0xA0;
...
+const errorNobleNotAlreadyScanning = 'No scan started';
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 4]
```diff
@@ -232,9 +233,13 @@ const obciByteStart = 0xA0;
...
+const errorNobleNotInPoweredOnState = 'Please turn blue tooth on.';
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 5]
```diff
@@ -749,6 +755,7 @@ module.exports = {
   },
   /** Possible Sample Rates */
   OBCISampleRate125: obciSampleRate125,
+  OBCISampleRate200: obciSampleRate200,
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 6]
```diff
@@ -758,9 +765,13 @@ module.exports = {
   OBCIByteStart: obciByteStart,
   OBCIByteStop: obciByteStop,
   /** Errors */
+  OBCIErrorNobleAlreadyScanning: errorNobleAlreadyScanning,
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 7]
```diff
@@ -758,9 +765,13 @@ module.exports = {
...
+  OBCIErrorNobleNotAlreadyScanning: errorNobleNotAlreadyScanning,
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: openBCIConstants.js [hunk 8]
```diff
@@ -758,9 +765,13 @@ module.exports = {
...
+  OBCIErrorNobleNotInPoweredOnState: errorNobleNotInPoweredOnState,
```
Comment (aj-ptw) [inline openBCIConstants.js]: remove

File: test/OpenBCIConstants-test.js [hunk 9]
```diff
@@ -765,6 +765,9 @@ describe('OpenBCIConstants', function () {
       assert.equal(125, k.OBCISampleRate125);
     });
     it('should be 250', function () {
+      assert.equal(200, k.OBCISampleRate200);
```
Comment (aj-ptw) [inline test/OpenBCIConstants-test.js]: remove

File: changelog.md [hunk 10]
```diff
@@ -1,3 +1,28 @@
...
+
+### Breaking Changes
+* Change name of `OpenBCIBoard` to `Cyton` to follow new naming convention.
+    Simply change:
+    ```ecmascript 6
+    const OpenBCIBoard = require('openbci').OpenBCIBoard;
+    const ourBoard = new OpenBCIBoard();
+    ```
+    ```ecmascript 6
+    const Cyton = require('openbci').Cyton;
```
Comment (teonbrooks) [inline changelog.md]: throwback name ;)

Comment (aj-ptw) [inline changelog.md]: How do you feel about it?

Comment (teonbrooks) [inline changelog.md]: I didn't know what it meant until I looked it up. It's shorter, which is nice, but less explicitly descriptive, which is less nice

**Pull request closed without merging**